reactive-banana/src/Reactive/Banana/Combinators.hs

```diff
@@ -349,6 +350,15 @@ split e = (filterJust $ fromLeft <$> e, filterJust $ fromRight <$> e)
     fromRight (Right b) = Just b
 
 
+-- | Keep only the next occurence of an event.
+--
```

Comment:
```suggestion
-- (Also aids the garbage collector by indicating that the result event can be discarded after its only occurrence.)
```

Reply:
Sounds good! I'm going to slightly reword this without parens, as it seems important enough to stand alone.